src/Main.hs

```diff
@@ -65,7 +74,11 @@ options = [
            "Maximum number of recursive bounces a ray is allowed before it is terminated",
     Option "o" ["output"]
            (ReqArg (\ f opts -> opts { output = f }) "FILE")
-           "Output file (should end in \"png\")"
+           "Output file (should end in \"png\")",
+    Option "g" ["gamma"]
+            (ReqArg (\ g opts -> opts { gamma = parsePositiveDouble g "gamma" })
+                   "BOUNCES")
```

Comment:
Should be `"GAMMA"` so that `cabal new-run ray-tracer -- --help` will use the correct place-holder name.  *e.g.* here's the current output:

```
Usage: ray-tracer [OPTION...]
  -s SAMPLES  --samples=SAMPLES  Number of anti-aliasing samples to take per pixel
  -b BOUNCES  --bounces=BOUNCES  Maximum number of recursive bounces a ray is allowed before it is terminated
  -o FILE     --output=FILE      Output file (should end in "png")
```